test_util/azure_test_driver.py

```diff
@@ -126,6 +131,34 @@ def poll_on_dcos_ui_up():
 
         print("Waiting for DC/OS UI at: {} ...".format(master_url))
         poll_on_dcos_ui_up()
+
+        # Run test now, so grab IPs
+        nmc = NetworkManagementClient(credentials, subscription_id)
+        ip_buckets = {
+            'masterNodeNic': [],
+            'slavePrivateNic': [],
+            'slavePublicNic': []}
+
+        for resource in rmc.resource_groups.list_resources(group_name):
+            for bucket_name, bucket in ip_buckets.items():
+                if resource.name.startswith(bucket_name):
+                    nic = nmc.network_interfaces.get(group_name, resource.name)
+                    all_ips = []
+                    for config in nic.ip_configurations:
+                        all_ips.append(config.private_ip_address)
+                    bucket.extend(all_ips)
+
+        with closing(SSHTunnel('core', 'ssh_key', master_lb, port=2200)) as t:
+            integration_test(
+                    tunnel=t,
+                    test_dir='/home/core',
+                    dcos_dns=master_lb,
+                    master_list=ip_buckets['masterNodeNic'],
+                    agent_list=ip_buckets['slavePrivateNic'],
+                    public_agent_list=ip_buckets['slavePublicNic'],
+                    provider='azure',
+                    test_dns_search=False,
```

Comment:
Ok, this makes sense now... when `test_dns_search` is set to `False` that is translated into the `dns_search_set` proxy class configuration here: https://github.com/dcos/dcos/blob/master/packages/dcos-integration-test/extra/integration_test.py#L330 ... and False is what we want for the Azure environment where the DNS search domain is not yet configurable.